### DISCUSSION TITLE
10.8.13

### DISCUSSION THREAD

**Discussion by joshuaboniface** [jellyfin/jellyfin#10647] [Announcements]
# Jellyfin 10.8.13

Stable hotfix release for 10.8.z release branch.

## New Features and Major Improvements

N/A

## Release Notes

* **NOTICE:** The customizable FFmpeg binary path in the WebUI/API has been REMOVED for security reasons. Please see [the corresponding blog post here](https://jellyfin.org/posts/jellyfin-security-and-you) for further details.

This bugfix release contains two GHSA security fixes; the details of these vulnerabilities **will be released publicly on December 5th 2023**. Please ensure you update your systems as soon as possible. Credit to @Kwstubbs (of GitHub Security Labs), @FredericLinn, and @mawalu for finding these.

## Changelog

GitHub Project: https://github.com/orgs/jellyfin/projects/29

### [jellyfin](https://github.com/jellyfin/jellyfin) [3]

Note: Dependabot & Renovate automatic PRs are excluded from this list.

 * #10625 [@ilovepilav] Fix: Plugin Installed Alerts missing from Admin Dashboard #10620
 * GHSA-rr9h-w522-cvmr [@joshuaboniface] Remove the functionality of /System/MediaEncoder/Path
 * GHSA-866x-wj5j-2vf4 [@crobibero] Validate codec and container

### [jellyfin-web](https://github.com/jellyfin/jellyfin-web) [3]

Note: Dependabot & Renovate automatic PRs are excluded from this list.

 * jellyfin/jellyfin-web#5019 [@joshuaboniface] Set FFmpeg path disabled
 * jellyfin/jellyfin-web#4993 [@rafma0] Remove FLAC Audio in Video Support for Tizen (Backport #4938)
 * jellyfin/jellyfin-web#4992 [@rafma0] Fix jittering in emby-checkbox (Backport #4929)

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin/releases/tag/v10.8.13'>10.8.13</a>.</em>

**Comment by jrbski** [comment]
Is there an issue with the ubuntu repo? I got an error about the filesize when I ran apt update: 

![image](https://github.com/jellyfin/jellyfin/assets/33815500/cfde2561-49fd-427b-8be8-d63113fb8517)

**Reply by jrbski** [reply]
Looks like I was too quick and mirrors did finally complete sync'ing 
![image](https://github.com/jellyfin/jellyfin/assets/33815500/6781a6af-84be-4014-84e2-8b10f4591085)

**Comment by Kavey78** [comment]


**Reply by Shadowghost** [reply]
We do not maintain any CentOS packages within repositories, so please contact their maintainer.

**Comment by DrewTheGiraffe** [comment]
Couldn't get either 10.8.12 or 10.8.13 to run correctly on windows 10 x64, had to revert back to 10.8.10, working fine again.
Issue seemed to be the program wouldn't load past the splash screen to the login screen. Tried on multiple computers and one windows server OS. Windows server 2022 on DELL R250. Same issue. Hope this makes it in front of a Dev or someone pushing a new PR.

**Comment by TrevorDunne** [comment]
10.8.13 Working fine here with Windows 2016 server and Firefox 122 on windows 10 client I did have to switch off my ad-blocker the controls at bottom of video vanished.
